### DISCUSSION TITLE
Cannot remove repo without img

### DISCUSSION THREAD

**Discussion by vv0rk0ffice** [klausmeyer/docker-registry-browser#691] [Q&A] [answered]
i had delete docker image, but the still shoewn

![Screenshot 2024-11-17 195245](https://github.com/user-attachments/assets/8940f37a-44a3-4f20-9c97-36be4d6c1ebc)

**Comment by klausmeyer** [comment, marked as answer]
Unfortunately the registry does not allow to run garbage collection via API.

You'll need to perform it manually inside the registry:

https://distribution.github.io/distribution/about/garbage-collection/

**Reply by vv0rk0ffice** [reply]
thx for your response. 

i will see if there a script to do this `garbage collection` or I'll  make one
